### ISSUE TITLE
Check that .checkbox and .radio markup is correctly structured

### ISSUE THREAD

**Issue by cvrebert** [twbs/bootlint#20]
Valid structure is:

``` css
div.checkbox>label>input[type="checkbox"]
div.radio>label>input[type="radio"]
label.checkbox-inline>input[type="checkbox"]
label.radio-inline>input[type="radio"]
```
